### PR TITLE
Update Docker and Docker Compose.

### PR DESCRIPTION
**Pull request by perylemke**
Hello,

Updating Docker and Docker Compose in BoxOS.

Regards,

### CODE REVIEW

File: initial.sh [hunk 1]
```diff
@@ -46,19 +46,29 @@ Debian)
...
+    ARCH="armhf"
+    ;;
+*)
+    echo "Unsupported arch."
+    ;;
+esac
+
+DOCKER_REPO="deb [arch=${ARCH}] https://download.docker.com/linux/${OS} ${VER_NAME} stable"
+DOCKER_COMPOSE_VERSION="1.19.0"
+#gpg_fingerprint="58118E89F3A912897C070ADBF76221572C52609D"
```
Comment (avelino) [inline initial.sh]: remove comment

**Review by avelino — CHANGES_REQUESTED** [review]

**Comment by perylemke** [comment]
Comment removed. Thanks :)

**Review by avelino — APPROVED** [review]
LGTM

**Review by crgimenes — APPROVED** [review]
LGTM

**Pull request merged**